app.js

```diff
@@ -74,82 +74,110 @@ app.use(function (req, res, next) {
+    res.setHeader('X-Hello-Human', 'You must be bored. You should work for us. Email jdorfman+theheader@maxcdn.com or @jdorfman on Twitter.');
     res.setHeader('Cache-Control', 'public, max-age=' + oneHourToSec);
     res.setHeader('Expires', new Date(Date.now() + oneHourToMilliSec).toUTCString());
     res.setHeader('Last-Modified', new Date().toUTCString());
-    res.setHeader('Strict-Transport-Security', 'max-age=16070400; includeSubDomains; preload');
     res.setHeader('Accept-Ranges', 'bytes');
-    res.setHeader('X-Content-Type-Options', 'nosniff');
-    res.setHeader('X-Frame-Options', 'DENY');
-    res.setHeader('X-XSS-Protection', '1; mode=block');
+
+    res.locals.nonce = Buffer.from(uuid.v4(), 'utf-8').toString('base64');
 
     next();
 });
 
-csp.extend(app, {
-    policy: {
-        directives: {
-            'default-src': ['\'none\''],
-            'script-src': [
-                '\'self\'',
-                '\'unsafe-inline\'',
-                'maxcdn.bootstrapcdn.com',
-                'www.google-analytics.com',
-                'code.jquery.com',
-                'platform.twitter.com',
-                'cdn.syndication.twimg.com',
-                'api.github.com'
-            ],
-            'style-src': [
-                '\'self\'',
-                '\'unsafe-inline\'',
-                'maxcdn.bootstrapcdn.com',
-                'fonts.googleapis.com',
-                'platform.twitter.com'
-            ],
-            'img-src': [
-                '\'self\'',
-                'data:',
-                'www.google-analytics.com',
-                'bootswatch.com',
-                'syndication.twitter.com',
-                'pbs.twimg.com',
-                'platform.twitter.com',
-                'analytics.twitter.com',
-                'stats.g.doubleclick.net'
-            ],
-            'font-src': [
-                '\'self\'',
-                'maxcdn.bootstrapcdn.com',
-                'fonts.gstatic.com'
-            ],
-            'manifest-src': ['\'self\''],
-            'frame-src': [
-                '\'self\'',
-                'platform.twitter.com',
-                'syndication.twitter.com',
-                'ghbtns.com'
-            ],
-            'child-src': [
-                '\'self\'',
-                'platform.twitter.com',
-                'syndication.twitter.com',
-                'ghbtns.com'
-            ],
-            'report-uri': [
-                'https://d063bdf998559129f041de1efd2b41a5.report-uri.io/r/default/csp/enforce'
-            ]
-        },
-        useScriptNonce: false,
-        useStyleNonce: false
+app.use(helmet({
+    dnsPrefetchControl: false,
+    frameguard: {
+        action: 'deny'
     }
-});
+}));
 
+app.use(helmet.hsts({
+    force: true,
+    includeSubdomains: true,
+    maxAge: 10886400,
+    preload: true
+}));
+
+app.use(helmet.contentSecurityPolicy({
```

Comment:
This down the road we should pull this out into it's own module... getting it out of app.js.